tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -65,7 +95,69 @@ public async Task SyncButton()
         // temporary, we'll need to give the users a way to disable this.
         // if they don't want to use hyper-v
         _notificationService.CheckIfUserIsAHyperVAdmin();
+
+        // Reset the sort and filter
+        SelectedSortIndex = -1;
+        Providers = new ObservableCollection<string> { _stringResource.GetLocalized("AllProviders") };
+        SelectedProviderIndex = 0;
+
+        // Reset the old sync timer
+        _cancellationTokenSource.Cancel();
+        await _dispatcher.EnqueueAsync(() => LastSyncTime = _stringResource.GetLocalized("MomentsAgo"));
+
         await LoadModelAsync();
+
+        // Start a new sync timer
+        _ = Task.Run(async () =>
+        {
+            await RunSyncTimmer();
+        });
+    }
+
+    // Updates the last sync time on the UI thread after set delay
+    private async Task UpdateLastSyncTimeUI(string time, TimeSpan delay, CancellationToken token)
+    {
+        await Task.Delay(delay, token);
+
+        if (!token.IsCancellationRequested)
+        {
+            await _dispatcher.EnqueueAsync(() => LastSyncTime = time);
+        }
+    }
+
+    private async Task RunSyncTimmer()
+    {
+        _cancellationTokenSource = new CancellationTokenSource();
+        var cancellationToken = _cancellationTokenSource.Token;
+
+        await UpdateLastSyncTimeUI(_stringResource.GetLocalized("MinuteAgo"), TimeSpan.FromMinutes(1), cancellationToken);
+        if (cancellationToken.IsCancellationRequested)
+        {
+            return;
+        }
+
+        // For the first 2-5 minutes, in 1 minute increments
+        for (var i = 2; i <= 5; i++)
+        {
+            await UpdateLastSyncTimeUI($"{i} {_stringResource.GetLocalized("MinutesAgo")}", TimeSpan.FromMinutes(1), cancellationToken);
+            if (cancellationToken.IsCancellationRequested)
+            {
+                return;
+            }
+        }
+
+        // For the 10-55 minutes, in 5 minute increments
+        for (var i = 2; i <= 11; i++)
+        {
+            await UpdateLastSyncTimeUI($"{i * 5} {_stringResource.GetLocalized("MinutesAgo")}", TimeSpan.FromMinutes(5), cancellationToken);
+            if (cancellationToken.IsCancellationRequested)
+            {
+                return;
+            }
+        }
+
+        // For an hour and more
+        await UpdateLastSyncTimeUI(_stringResource.GetLocalized("HourAgo"), TimeSpan.FromMinutes(5), cancellationToken);

```

Comment:
`TimeSpan.FromMinutes(5),` Might be a missed copy/paste from above but seems like you meant to use hours here and not minutes here right?

Also I believe we haven't covered the case for showing "Hours Ago"  and the Day/Days case as well

Reply:
No, I meant to use minutes. Updated the description to reflect the current design.